### DISCUSSION TITLE
Account Blocked without reason.

### DISCUSSION THREAD

**Discussion by sgalvao** [vercel/community#3628] [Account Issue] [answered]
### Question

I received an email saying that my vercel account was blocked, I would like to recover my account because the team connected to it has been offline for 24 hours and I have not received any explanations about anything related to my ban.
I'm a Pro user and I've been using Vercel for years and I've never had a problem like this!! I'm losing my customers and money because of this meaningless lockdown.

**Comment by amyegan** [comment]
Sorry you're having account issues. Due to the sensitive nature of account info, we handle these issues in support tickets. Please email [registration@vercel.com](mailto:registration@vercel.com) or [submit a case](https://vercel.com/help#issues) to the Customer Success Team. Either option will create a new support ticket. 

Vercel employs multiple layers of security and verification, so be sure to include the email and username for the account in your message. Providing additional information about your account and the project you are deploying can help accelerate the process.

**Reply by sgalvao** [reply]
I can't submit a case because my account is blocked, so I can't login to submit, what is the faster option to do?

**Reply by amyegan** [reply, marked as answer]
Please email [registration@vercel.com](mailto:registration@vercel.com) to create a new case.
